pkg/runner/cluster_k8s.go

```diff
@@ -146,67 +148,156 @@ func (*ClusterK8sRunner) Run(ctx context.Context, input *api.RunInput, ow io.Wri
 
 	// Define k8s client configuration
 	config := defaultKubernetesConfig()
-	k8scfg, err := clientcmd.BuildConfigFromFlags("", config.KubeConfigPath)
-	if err != nil {
-		return nil, fmt.Errorf("could not start k8s client from config: %v", err)
-	}
 
-	// Create the clientset
-	clientset, err := kubernetes.NewForConfig(k8scfg)
+	workers := 20
```

Comment:
Scheduling all pods to Kubernetes and then pooling for state is faster when we use a few clients concurrently.

Reply:
Great discovery!